lib/features/copy-paste/BpmnCopyPaste.js

```diff
@@ -60,23 +59,62 @@ export default function BpmnCopyPaste(
     ]);
 
     if (businessObject.default) {
-      descriptor.default = businessObject.default.id;
+      descriptor.$default = businessObject.default.id;
```

Comment:
Why `$default`? :thinking:

Reply:
I guess I wanted to clarify that this is a model element property whereas parent, source and target, for example, are properties of the diagram element. We can just name it `default`.